core/src/main/scala/kafka/server/KafkaBroker.scala

```diff
@@ -90,7 +92,7 @@ trait KafkaBroker extends KafkaMetricsGroup {
     explicitMetricName(KafkaBroker.metricsPrefix, KafkaBroker.metricsTypeName, name, metricTags)
   }
 
-  newGauge("BrokerState", () => brokerState.get.value())
+  newGauge("BrokerState", () => brokerState.value)
```

Comment:
(this is not related to this PR) it is a bit weird to me that we expose "byte" as "broker state".

Reply:
Right. To change this, it would require a KIP.